pom.xml

```diff
@@ -1341,6 +1341,27 @@
   </modules>
 
   <profiles>
+    <!-- Profile to include CMIS datastorage -->
+    <profile>
+      <id>datastorage-cmis</id>
+      <modules>
+        <module>datastorages/cmis</module>
```

Comment:
This looks incorrect and would result in a strange / unstable build.

* `pom.xml` includes module `datastorages/cmis`
` datastorages/pom.xml` includes `cmis`

Recommendation:

* Remove the profiles from here and added to `datastores/pom.xml`

Reply:
Done: https://github.com/geonetwork/core-geonetwork/pull/7302/commits/66cf35912aa100a1ade84cc685dc2d7c4542f10b